docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -256,36 +261,37 @@ runtime as well as in the :ref:`cluster configuration
 is used to avoid a situation in which host failures cascade throughout the cluster as load
 increases.
 
-Panic thresholds work in conjunction with priorities. If number of healthy hosts in a given priority
-goes down, Envoy will try to shift some traffic to lower priorities. If it succeeds in finding enough 
-healthy hosts in lower priorities, Envoy will disregard panic thresholds. In mathematical terms, 
-if normalized total health across all priority levels is 100%, Envoy disregards panic thresholds but continues to
-distribute traffic load across priorities according to algorithm described :ref:`here <arch_overview_load_balancing_priority_levels>`. 
-However, when value of normalized total health drops below 100%, Envoy assumes that there is not enough healthy
-hosts across all priority levels. It continues to distribute traffic load across priorities, but if a specific priority level's
-health is below panic threshold, traffic will go to all hosts in that priority regardless of their health.
+Panic thresholds work in conjunction with priorities. If the number of healthy hosts in a given
+priority goes down, Envoy will try to shift some traffic to lower priorities. If it succeeds in
+finding enough healthy hosts in lower priorities, Envoy will disregard panic thresholds. In
+mathematical terms, if normalized total health across all priority levels is 100%, Envoy disregards
+panic thresholds and continues to distribute traffic load across priorities according to the
```

Comment:
> since we don't change how we distribute load between priorities even if we respect panic thresholds

That is true, and I am not sure if we should talk about panic thresholds here or not. But the fact is that panic is part of the whole picture. I will work on #4685 and will have to update this section anyway. I will ask both of you @snowp @fredlas to review docs after my changes.

Reply:
Yeah, given that the way this work is likely to change soon I wouldn't spend too much time word smithing this part.

Reply:
I didn't do any actual rewriting in this paragraph: just line breaks and typos. So, would you prefer if I just reverted these changes?